lib/message_bus.rb

```diff
@@ -554,6 +554,25 @@ def keepalive_interval
     @config[:keepalive_interval] || 60
   end
 
+  # Registers a client message filter that allows messages to be filtered from the client.
+  #
+  # @param [String] channel_prefix channel prefix to match against a message's channel
```

Comment:
technically we are allowing String or Regexp

Reply:
O I just ran into this but if we support Ruby 2.4, we can't pass it a Regexp

Reply:
no probs